keyboards/alhenkb/macropad5x4/info.json

```
@@ -0,0 +1,52 @@
+{
+    "manufacturer": "AlhenKB",
+    "keyboard_name": "Macropad 5x4",
+    "maintainer": "alhenx",
+    "bootloader": "caterina",
```

This sets the processor and bootloader option, and makes it easier for converters
```suggestion
    "development_board": "promicro",
```